README.md

```diff
@@ -124,87 +118,85 @@ https://github.com/azjezz/benchmark-php-routing/actions
 
 ```sh
-+------------------------------- Benchmark results for group cached. -------------+-----------------+
-| Case                                 | Scenario     | Routes | Time             | Per Second      |
-+--------------------------------------+--------------+--------+------------------+-----------------+
-| hack-routing:cached(apcu)            | benchAll     | 364    | 0.062711 seconds | 5804.4042900212 |
-| hack-routing:cached(apcu)            | benchLast    | 300    | 0.058096 seconds | 5163.8727469713 |
-| hack-routing:cached(apcu)            | benchLongest | 300    | 0.091959 seconds | 3262.3234397362 |
-| hack-routing:cached(file)            | benchLast    | 300    | 0.114034 seconds | 2630.7957674481 |
-| hack-routing:cached(file)            | benchAll     | 364    | 0.139894 seconds | 2601.969902413  |
-| fast-route(mark):cached(file)        | benchLast    | 300    | 0.134784 seconds | 2225.7798155401 |
-| fast-route(group_pos):cached(file)   | benchLast    | 300    | 0.138804 seconds | 2161.3216394761 |
-| fast-route(group_pos):cached(file)   | benchAll     | 364    | 0.172261 seconds | 2113.0726088731 |
-| fast-route(char_count):cached(file)  | benchLast    | 300    | 0.143844 seconds | 2085.5908745852 |
-| fast-route(char_count):cached(file)  | benchAll     | 364    | 0.177352 seconds | 2052.4160953288 |
-| fast-route(mark):cached(file)        | benchAll     | 364    | 0.190084 seconds | 1914.9430632032 |
-| fast-route(group_count):cached(file) | benchLast    | 300    | 0.157413 seconds | 1905.8145698975 |
-| hack-routing:cached(file)            | benchLongest | 300    | 0.160584 seconds | 1868.1814537561 |
-| fast-route(mark):cached(file)        | benchLongest | 300    | 0.172731 seconds | 1736.8054041072 |
-| fast-route(group_count):cached(file) | benchAll     | 364    | 0.209652 seconds | 1736.2109208683 |
-| fast-route(group_pos):cached(file)   | benchLongest | 300    | 0.177259 seconds | 1692.4390499786 |
-| fast-route(group_count):cached(file) | benchLongest | 300    | 0.178499 seconds | 1680.6818409492 |
-| fast-route(char_count):cached(file)  | benchLongest | 300    | 0.182353 seconds | 1645.1605817657 |
-| symfony:cached(file)                 | benchAll     | 364    | 0.235754 seconds | 1543.9837625136 |
-| symfony:cached(file)                 | benchLast    | 300    | 0.197746 seconds | 1517.0973960914 |
-| symfony:cached(file)                 | benchLongest | 300    | 0.215139 seconds | 1394.447879888  |
-+--------------------------------------+--------------+--------+------------------+-----------------+
++------------------------------ Benchmark results for group cached. -------------+-----------------+
+| Case                          | Scenario           | Routes | Time             | Per Second      |
++-------------------------------+--------------------+--------+------------------+-----------------+
+| fast-route(mark):cached(file) | benchAll           | 364    | 0.004870 seconds | 74744.279643592 |
```

Comment:
In my laptop SF router handles this in 0.002404 seconds vs 0.003114 seconds of FastRoute

Reply:
i would consider this to be more consistent, but we can investigate that later, i would do a run tomorrow and see